core/src/sql/statements/live.rs

```diff
@@ -153,3 +155,125 @@ impl InfoStructure for LiveStatement {
 		})
 	}
 }
+
+#[cfg(test)]
```

Comment:
@amankrx would it be possible to move these tests into the `lib/tests` folder? This way we can keep the functionality of the `sql` module cleaner, and not focused around implementation details (we are splitting out the modules to make this cleaner as we go forward).

Reply:
I had initially kept it in the `lib/tests` folder, but @phughk suggested that it should be better off as a unit test here.

Reply:
We can move this to `kvs/tests` though.